<?xml version="1.0"?>
<launch>
    <node name="moveit_manual" pkg="ROS_UI_backend" type="manual_control.py" output="screen">
    </node>
</launch>  
